lib/WeBWorK3/Hooks.pm

```diff
@@ -19,6 +20,7 @@ our $exception_handler = sub {
 			$output->{message} = $_->message
 				if (ref($_) && (ref($_) eq 'Mojo::Exception' || ref($_) =~ /^DB::Exception/x));
 			$output->{message} = $_ if ($_ && ref($_) eq 'DBIx::Class::Exception');
+			$log->error($output->{message});
```

Comment:
There is no need to `use Mojo::Log` in this file and create a new instance of that object.  Just use the already existing application log instance.  For that change this to
`$c->app->log->error($output->{message});`

Reply:
I think you can shorten that to `$c->log->error(...)`

Reply:
Yep.  It looks like you can.